freqtrade/plugins/pairlist/AgeFilter.py

```diff
@@ -64,7 +64,7 @@ def filter_pairlist(self, pairlist: List[str], tickers: Dict) -> List[str]:
         since_ms = int(arrow.utcnow()
                        .floor('day')
                        .shift(days=-self._min_days_listed - 1)
-                       .float_timestamp) * 1000
+                       .int_timestamp) * 1000
```

Comment:
By using int_timestamp, you can also omit the `int()` - as now it's kinda pointless.

Obviously applies to all occurances ...

Reply:
You are right about it, I overlooked that! 😁